### PR TITLE
This should add all the audio back except for the lines causing the error from before

### PR DESCRIPTION
**Pull request by outercloudstudio**
## Description
It added back the removed lines of code except for the one in base window.

## Motivation and Context
This fixed the weird circle dependencies errors being caused.

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Checklist:
- [x] My code follows the code style of this project.
- [x] I have tested my changes and confirmed that they are working

### CODE REVIEW

File: src/components/Windows/LoadingWindow/LoadingWindow.ts [hunk 1]
```diff
@@ -15,11 +15,13 @@ export class LoadingWindow extends BaseWindow {
 	}
 
 	open(message?: string) {
+		//new Audio('/audio/click5.ogg').play()
```
Comment (solvedDev) [inline src/components/Windows/LoadingWindow/LoadingWindow.ts:18]: Can we remove the comments in this file and replace them with `// TODO: Add sounds`?

File: src/components/Windows/UnsavedFile/UnsavedFile.ts [hunk 2]
```diff
@@ -18,19 +18,22 @@ export class UnsavedFileWindow extends BaseWindow<'cancel' | 'close' | 'save'> {
 	}
 
 	async noSave() {
+		//new Audio('/audio/click5.ogg').play()
```
Comment (solvedDev) [inline src/components/Windows/UnsavedFile/UnsavedFile.ts:21]: Same here.

**Review by solvedDev — CHANGES_REQUESTED** [review]

**Comment by outercloudstudio** [comment]
This should fix it.

**Review by solvedDev — APPROVED** [review]
LGTM, thanks 👏

**Pull request merged**